src/include/truncate.h

```diff
@@ -0,0 +1,26 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WT_TRUNCATE_INFO
+ *	A set of context associated with a range truncate operation.
+ */
+struct __wt_truncate_info {
+    WT_SESSION_IMPL *session;
+    const char *uri;
+    WT_CURSOR *start;
+    WT_CURSOR *stop;
+    WT_ITEM *orig_start_key;
+    WT_ITEM *orig_stop_key;
+
+/* AUTOMATIC FLAG VALUE GENERATION START 0 */
+#define WT_TRUNC_EXPLICIT_START 0x1u
+#define WT_TRUNC_EXPLICIT_STOP 0x2u
+    /* AUTOMATIC FLAG VALUE GENERATION STOP 32 */
+    uint32_t flags;
```

Comment:
Is 8 bit enough?

Reply:
It is, but I tend to default to 32 bit values unless there is a reason to be space sensitive, which I don't think there is in this case.
If you feel strongly, I can change it.

Reply:
No, I don't.